bootstrap.sh

```diff
@@ -10,6 +10,9 @@ cd /vagrant
 # Install dependencies with Pipenv
 pipenv sync --dev
 
+# Run database migrations
+pipenv run python manage.py migrate
+
 # run our app. Nohup and “&” are used to let the setup script finish
 # while our app stays up. The app logs will be collected in nohup.out
 nohup pipenv run python manage.py runserver 0.0.0.0:8000 &
```

Comment:
Please update this line as per the discussion in os-07 slack channel

Reply:
@DeanBiton Hi, edit this file according to [Barak's commit](https://github.com/redhat-beyond/JobSeeker/issues/41#issuecomment-1073349042).
Once you do it, please state in this PR's description:
```
Close #41
```